README.md

```diff
@@ -1,2 +1,68 @@
 # EmbeddedChat
-An easy to use full-stack component  (ReactJS + backend behaviors)  embedding Rocket.Chat into your webapp
+
+An easy to use full-stack component (ReactJS + backend behaviors) embedding Rocket.Chat into your webapp.
+
+*EmbeddedChat is a full-stack React component node module of the RocketChat application that is fully configurable, extensible, and flexible for use. It is tightly bound with the RocketChat server using Rocket.Chat nodejs SDK and its UI using RocketChat's Fuselage Design System.*
+
+![embeddedchatwall](https://user-images.githubusercontent.com/73601258/178119162-ecabb9b7-e3ae-4c70-8ab2-f6c02856f4c6.png)
```

Comment:
Not a good screen capture for this purpose.

It looks no different than a non-embedded chat window.    Bringing up the bigger question of -- what is this project for?

Your screen capture should clearly show (ALL) the added values that this component bring to our community.    This picture *is* your THOUSAND WORDS.

Reply:
True. 100% agreed. Let me search a bit around for mockups that I can use. I will send some samples in the channel.